pyproject.toml

```diff
@@ -56,9 +56,9 @@ pep8 = "^1.7"
 pydocstyle = "^6.1.1"
 pylint = "^2.4"
 pytest = "*"
-pytest-cov = "*"
+pytest-cov = "4.1.0"
```

Comment:
Pinning to a specific version is okay, but should we do `^4.1.0` instead as we probably want to keep `pytest` and `coverage` up-to-date?